doc/source/cluster/aws-tips.rst

```diff
@@ -73,10 +95,199 @@ You should see something like
 Please refer to `this discussion <https://github.com/ray-project/ray/issues/9327>`__ for more details.
 
 
+.. _aws-cluster-efs:
+
+Amazon EFS
+~~~~~~~~~~
+
+To use Amazon EFS, install some utilities and mount the EFS in ``setup_commands``. Note that these instructions only work if you are using the AWS Autoscaler.
```

Comment:
Is there some EFS documentation we can link to here about what utilities we're installing and why? maybe this page? https://docs.aws.amazon.com/efs/latest/ug/installing-amazon-efs-utils.html#installing-other-distro